src/highlight/highlight.controller.ts

```
@@ -0,0 +1,52 @@
+export class HighlightController {
+    constructor(private readonly highlightService: HighlightService) { }
+
+    @Post()
+    async create(@Body() body: CreateHighlightDto) {
+        return this.highlightService.create(body);
+    }
+
+    @Patch(':project/:timestamp')
+    async update(
+        @Param('project') project: string,
+        @Param('timestamp') timestamp: Number,
```

Biome

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)